chat.tox.qTox.json

```diff
@@ -0,0 +1,44 @@
+{
+	"app-id": "chat.tox.qTox",
+	"runtime": "org.kde.Platform",
+	"sdk": "org.kde.Sdk",
+	"runtime-version": "5.10",
+	"command": "qtox",
+	"finish-args": [
+		"--share=network",
+		"--socket=pulseaudio",
+		"--socket=wayland",
+		"--socket=x11",
+		"--filesystem=host",
+		"--device=all"
+	],
+	"build-options": {
+		"cflags": "-O3 -DSQLITE_HAS_CODEC",
+		"cxxflags": "-O3"
+	},
+	"modules": [
+		{
+			"name": "sqlcipher",
```

Comment:
This needs an autoreconf in order to build for aarch64. example here: https://github.com/flathub/shared-modules/blob/08e9b9eead7a95139be5d1fbc24bea730fb78f7a/SDL/SDL_net-1.2.8.json